sway-core/src/language/parsed/expression/match_branch.rs

```diff
@@ -1,10 +1,33 @@
-use sway_types::span;
+use sway_types::{span, Span};
 
-use super::{Expression, Scrutinee};
+use super::{Expression, Literal, Scrutinee};
 
 #[derive(Debug, Clone)]
 pub struct MatchBranch {
     pub scrutinee: Scrutinee,
     pub result: Expression,
     pub(crate) span: span::Span,
 }
+
+impl MatchBranch {
+    pub fn catch_all(result: Expression) -> Self {
```

Comment:
Also these two functions, are they ever used? Sorry for being picky about this, but since they are `pub` I assume they will not be reported as dead by Clippy and the rustc.

Reply:
Removed.